### PR TITLE
[OSCD] - Force winword.exe to load DLL (LoLBin)

### PR DESCRIPTION
**Pull request by stvetro**
Task 20 from #1014

### CODE REVIEW

File: rules/windows/process_creation/win_winword_dll_load.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+references:
+    - https://github.com/LOLBAS-Project/LOLBAS/blob/master/yml/LOLUtilz/OtherMSBinaries/Winword.yml
+author: Victor Sergeev, oscd.community
+date: 2020/10/09
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    image_path:
+        Image|endswith: 'winword.exe'
```
Comment (yugoslavskiy) [inline rules/windows/process_creation/win_winword_dll_load.yml:14]: could you please add `\` here as well?

Comment (stvetro) [reply rules/windows/process_creation/win_winword_dll_load.yml]: done

Comment (yugoslavskiy) [reply rules/windows/process_creation/win_winword_dll_load.yml]: thank you!

**Comment by yugoslavskiy** [comment]
Thank you for your contribution, @stvetro! Great job!

**Review by yugoslavskiy — APPROVED** [review]
@thomaspatzke @Neo23x0 guys, I believe that this PR could be merged (:
please let us know what do you think about it if there is anything you would like to change.

**Pull request merged**